src/styles/calendar.scss

```diff
@@ -0,0 +1,51 @@
+* {
+  transition: background-color 0.5s, transform 1s;
```

Comment:
Applying styles to all elements can lead to unnecessary performance overhead. The browser has to process the styles for every element, even if they don't need those styles.
Move it to the specific selector.